packages/core/http/core-http-server-internal/src/http_server.ts

```diff
@@ -526,6 +527,8 @@ export class HttpServer {
       xsrfRequired: route.options.xsrfRequired ?? !isSafeMethod(route.method),
       access: route.options.access ?? (route.path.startsWith('/internal') ? 'internal' : 'public'),
     };
+    // Log HTTP API target consumer.
+    optionsLogger.debug(`access [${kibanaRouteOptions.access}] for path [${route.path}]`);
```

Comment:
mind that @jloleysens's [comment](https://github.com/elastic/kibana/pull/152621/files/bf7a3c36cb4394b06a3b3fa09a0553c3f958bfed#r1126120651) also suggested adding the method

Reply:
Apologies, adding that now.

Reply:
done: commit [fb86412](https://github.com/elastic/kibana/pull/152621/commits/fb86412ba872c2eccda4f106e9e8577a039da940)